woocommerce/woocommerce-ios
# [Blaze] Remove old endpoint usage

Part of: #12120 


## Description

- Remove old Blaze GET campaigns endpoints-related code. 
- Use new `BlazeCampaignListItem` in product form view to show "Promote with Blaze" button. 

📔 There is an issue with handling the values for the `status` field. The v1.1. sends different values than the mobile app expects. We have fixed this in #12207

## Testing instructions
- Replace a build from branch `trunk` with a build from this branch.
- Launch the app and the app launches without any core data related crashes.
- Login into a Woo store with Blaze campaigns.
- Navigate to the Products tab and tap on a published product.
- You should see the "Promote with Blaze" button.

## Screenshots

![Simulator Screen Recording - iPhone 15 Pro Max - 2024-03-04 at 19 08 36](https://github.com/woocommerce/woocommerce-ios/assets/524475/640576a3-4a57-43cc-b148-3c75bbfb8de4)


---
